resources/views/account/accept/create.blade.php

```diff
@@ -64,6 +64,15 @@
                             </label>
 
                         </div>
+                        <div class="col-md-12">
+                            <br>
+                            <div class="col-md-12" style="display:block;">
+                                <label id="note_label" for="note" style="text-align:center;" >{{trans('admin/settings/general.decline_message')}}</label>
```

Comment:
Whether we decide to always have the textarea showing or hidden then shown upon selection this message needs to be updated:

![CleanShot 2024-03-21 at 14 49 11@2x](https://github.com/snipe/snipe-it/assets/1141514/e2580cf3-35cf-41f4-8503-32aed39dce62)